packages/vite/src/node/optimizer/scan.ts

```
@@ -347,8 +348,6 @@ function globEntries(pattern: string | string[], environment: ScanEnvironment) {
         ? []
         : [`**/__tests__/**`, `**/coverage/**`]),
     ],
-    absolute: true,
-    suppressErrors: true, // suppress EACCES errors
```

Such errors are suppressed by default in `tinyglobby`.